src/main/java/ch/uzh/ifi/hase/soprafs24/entity/Tile.java

```diff
@@ -0,0 +1,51 @@
+package ch.uzh.ifi.hase.soprafs24.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "TILE")
+public class Tile implements Serializable {
+
+    @Id
+    @Getter
+    @Setter
+    // UUID 4 string
+    private String tileID;
```

Comment:
since it's inside the tile class id is enough. The tile prefix isn't necessary
same in dto